### PR TITLE
Feature/dev 4215f

### PR DESCRIPTION
**Pull request by Atash3000**

### CODE REVIEW

**Comment by adamdoe** [comment]
@Atash3000 - Please add testing steps here.

File: packages/chart/src/components/BrushChart.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        const leftHandlePath = 'M0.5,10A6,6 0 0 1 6.5,16V14A6,6 0 0 1 0.5,20ZM2.5,18V12M4.5,18V12'
+        const rightHandlePath = 'M6.5,10A6,6 0 0 0 0.5,16V14A6,6 0 0 0 6.5,20ZM4.5,18V12M2.5,18V12'
+        handleGroup
+          .append('text')
+          .attr('x', d => (d.side === 'left' ? 0 : -textWidth))
+          .attr('y', 30)
+          .text(d => (d.side === 'left' ? firstDate : lastDate))
+          .attr('font-size', '13px')
+
+        // handleGroup // ignore
```
Comment (adamdoe) [inline packages/chart/src/components/BrushChart.tsx:53]: Can we remove this unused code?

File: packages/chart/src/components/BrushChart.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  }
+  const handleMouseLeave = () => {
+    // hide tooltip text if brush was triggered
+    if (brushState.selection[0] !== 0 || brushState.selection[1] !== xMax) {
+      showTooltip(false)
+    }
+    showTooltip(false)
+  }
+
+  let brushHandle = (g, selection, firstDate, lastDate) => {
```
Comment (adamdoe) [inline packages/chart/src/components/BrushChart.tsx:36]: Should this be const?

File: packages/chart/src/components/EditorPanel/EditorPanel.tsx [hunk 3]
```diff
@@ -1958,7 +1958,7 @@ const EditorPanel = () => {
                         }
                         updateField={updateField}
                       />
-                      {false && visHasBrushChart && (
+                      {true && visHasBrushChart && (
```
Comment (adamdoe) [inline packages/chart/src/components/EditorPanel/EditorPanel.tsx:1961]: If we aren't hiding the brush anymore lets remove the true statement here.

**Review by adamdoe — CHANGES_REQUESTED** [review]

File: packages/chart/src/components/EditorPanel/EditorPanel.tsx (hunk 3 above)
Comment (Atash3000) [reply packages/chart/src/components/EditorPanel/EditorPanel.tsx:1961]: Fixed!

File: packages/chart/src/components/BrushChart.tsx (hunk 2 above)
Comment (Atash3000) [reply packages/chart/src/components/BrushChart.tsx:36]: Fixed!

**Pull request merged**